.circleci/regenerate.py

```diff
@@ -49,6 +49,10 @@ def build_workflows(prefix="", upload=False, filter_branch=None, indentation=6):
                     ):
                         # the fields must match the build_docs "requires" dependency
                         fb = "/.*/"
+
+                    if os_type == "macos" and btype == "wheel":
+                        continue
```

Comment:
Can we perform all the `continue` at the beginning at the same place as the existing one? (where "rocm" is checked)